src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -980,4 +1029,4 @@ public InputStream getFileContent(BitbucketSCMFile file) throws IOException, Int
                 .expand();
         return getRequestAsInputStream(url);
     }
-}
+}
```

Comment:
Please follow project standard and do not delete final new line

Reply:
I must have misunderstood the results of the checks - thought it was complaining about that :-/ Will fix, sorry